deploy/ansible/roles/goshimmer-node/templates/docker-compose-analysis-server.yml.j2

```diff
@@ -0,0 +1,31 @@
+#jinja2: trim_blocks:True, lstrip_blocks:True
```

Comment:
It's true for now. But it will change with the next PR when I will need to use variables for host URLs. So let's keep as template to not move it back and forth